app/styles/layout/_select-inline.scss

```diff
@@ -204,6 +204,7 @@ $list-height: 220px;
       margin: 5px 7px 0 0;
       position: absolute;
       width: $icon-width;
+      left: 10px;
```

Comment:
This is definitely the right fix.  I did a codepen because I could not figure out why: http://codepen.io/chjpugh/pen/yMJLZw

Basically, because the left is not set, the spec is to make it auto.  For Firefox, that means treating this basically as a relative div (so the float affects it).  For Chrome, it for some reason adds the left = 10px.

I took a screenshot of both browser's compiling-
Firefox: https://cl.ly/2h3z2Y0i3i3b
Chrome: https://cl.ly/2t2n1P2r1a1N